.github/workflows/weekly-comprehensive.yml

```
@@ -0,0 +1,169 @@
+      run-integration-tests: true
+      run-security-tests: true
+      enable-codecov: false
+
+  test-python-314:
+    name: CI (Python 3.14 - Experimental)
+    uses: ByronWilliamsCPA/.github/.github/workflows/python-ci.yml@main
+    permissions:
+      contents: read
+      pull-requests: write
+      checks: write
+    continue-on-error: true  # Allow 3.14 to fail (pyarrow compatibility)
```

actionlint (1.7.10)

when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "test-python-314"

(syntax-check)